src/Pact/Analyze/Eval/Core.hs

```diff
@@ -379,6 +379,9 @@ evalCore (ObjTake ty@(SObjectUnsafe schema') _keys obj) = withSing ty $ do
   case sing @a of
     SObjectUnsafe schema -> pure $ S prov $ evalDropTake (obj' :< schema') schema
 
+evalCore (ObjLength (SObjectUnsafe (SingList hlist)) _obj) = pure $
+  getSum $ foldHList (\_ _ -> Sum 1) hlist
```

Comment:
do SBV Hlists not track length?

Reply:
No, they don't. We define this `HList` in `Pact.Analyze.Types.Types`:

https://github.com/kadena-io/pact/blob/6848dbd9ca0bef015af9980c7298ffbf3802d1b6/src/Pact/Analyze/Types/Types.hs#L91-L97